scripts/migrations/015_migrate_project_field_value.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python
```

Comment:
This script needs to be mentioned and explained in the `README.md`

Reply:
Thanks @maxhbr for suggestion.
Have mentioned and explained the script in README.md